src/com/facebook/buck/cli/DownloadConfig.java

```diff
@@ -52,13 +53,13 @@ public DownloadConfig(BuckConfig delegate) {
     return delegate.getValue("download", "maven_repo");
   }
 
-  public ImmutableSet<String> getAllMavenRepos() {
-    ImmutableSortedSet.Builder<String> repos = ImmutableSortedSet.naturalOrder();
-    repos.addAll(delegate.getEntriesForSection("maven_repositories").values());
+  public ImmutableMap<String, String> getAllMavenRepos() {
+    ImmutableSortedMap.Builder<String, String> repos = ImmutableSortedMap.naturalOrder();
```

Comment:
After this change, you could have multiple repos with the same URI but different aliases. Should I perhaps use a `BiMap` instead?